### DISCUSSION TITLE
feat: add ignore/exclude list for the linter

### DISCUSSION THREAD

**Discussion by pums974** [krvajal/vscode-fortran-support#704] [Q&A - deprecated see Fortran Discourse] [answered]
### Is there an existing request for this?

- [X] I have searched the existing issues

### Feature Request

I would like to exclude some folder for the linter, eg:
 * tmp files
 * cmake files
 * documentation (ford)
 * ....

**Comment by gnikit** [comment, marked as answer]
For Q&A please use the Fortran-lang Discourse https://fortran-lang.discourse.group/c/help/vscode/16 or open a Discussion. The linter does not include anything by default. I suspect you are witnessing the effects of the initialisation process which picks up all the source files in your workspace. You can disable linter initialisation and control the include paths for the linter.

**Comment by pums974** [comment]
You are right, sorry I missed this option.
